contracts/lib/Errors.sol

```diff
@@ -128,7 +128,34 @@ library Errors {
     error LicenseRegistry__NotTransferable();
     /// @notice emitted on constructor if dispute module is not set
     error LicenseRegistry__ZeroDisputeModule();
-
+    error LicenseRegistry__IndexOutOfBounds(address ipId, uint256 index);
+    error LicenseRegistry__UnregisteredLicenseTemplate(address licenseTemplate);
+    error LicenseRegistry__NotLicenseTemplate(address licenseTemplate);
+    error LicenseRegistry__IpExpired(address ipId);
+    error LicenseRegistry__OriginalIpExpired(address ipId);
+    error LicenseRegistry__LicenseTermsNotExists(address licenseTemplate, uint256 licenseTermsId);
+    error LicenseRegistry__OriginalIpHasNoLicenseTerms(address ipId, uint256 licenseTermsId);
+    error LicenseRegistry__NoOriginalIp();
+    error LicenseRegistry__DerivativeIpAlreadyHasLicense(address derivativeIpId);
+    error LicenseRegistry__DerivativeAlreadyRegistered(address derivativeIpId);
+    error LicenseRegistry__OriginalIpTagged(address ipId);
+    error LicenseRegistry__DerivativeIsOriginal(address ipId);
+    error LicenseRegistry__OriginalIpUnmachedLicenseTemplate(address ipId, address licenseTemplate);
+
+    ////////////////////////////////////////////////////////////////////////////
+    //                            LicenseNFT                                  //
```

Comment:
Thought we are calling it License Token?

Reply:
This is the contract name of LicenseNFT. as LicenseToken is an NFT and aka LNFT. the contract name "LicenseNFT" should be OK.